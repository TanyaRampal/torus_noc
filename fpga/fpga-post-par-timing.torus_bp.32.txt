Copyright 1986-2022 Xilinx, Inc. All Rights Reserved.
-----------------------------------------------------------------------------------------
| Tool Version : Vivado v.2022.1 (lin64) Build 3526262 Mon Apr 18 15:47:01 MDT 2022
| Date         : Fri Dec  6 14:37:21 2024
| Host         : ECEUBUNTU2 running 64-bit Red Hat Enterprise Linux release 8.10 (Ootpa)
| Command      : report_timing -file fpga-post-par-timing.torus_bp.32.txt
| Design       : torus_bp
| Device       : 7z020-clg400
| Speed File   : -1  PRODUCTION 1.12 2019-11-22
-----------------------------------------------------------------------------------------

Timing Report

Slack (MET) :             0.786ns  (required time - arrival time)
  Source:                 ys[3].xs[2].torus_switch_xy/e_out_x_reg_reg[1]/C
                            (rising edge-triggered cell FDRE clocked by clk  {rise@0.000ns fall@3.000ns period=6.000ns})
  Destination:            ys[3].xs[3].torus_switch_xy/e_out_y_reg_reg[1]/D
                            (rising edge-triggered cell FDRE clocked by clk  {rise@0.000ns fall@3.000ns period=6.000ns})
  Path Group:             clk
  Path Type:              Setup (Max at Slow Process Corner)
  Requirement:            6.000ns  (clk rise@6.000ns - clk rise@0.000ns)
  Data Path Delay:        5.072ns  (logic 1.107ns (21.825%)  route 3.965ns (78.175%))
  Logic Levels:           3  (LUT3=1 LUT5=2)
  Clock Path Skew:        -0.049ns (DCD - SCD + CPR)
    Destination Clock Delay (DCD):    0.924ns = ( 6.924 - 6.000 ) 
    Source Clock Delay      (SCD):    0.973ns
    Clock Pessimism Removal (CPR):    0.000ns
  Clock Uncertainty:      0.035ns  ((TSJ^2 + TIJ^2)^1/2 + DJ) / 2 + PE
    Total System Jitter     (TSJ):    0.071ns
    Total Input Jitter      (TIJ):    0.000ns
    Discrete Jitter          (DJ):    0.000ns
    Phase Error              (PE):    0.000ns

    Location             Delay type                Incr(ns)  Path(ns)    PBlock               Netlist Resource(s)
  -------------------------------------------------------------------    ----------------------------------------
                         (clock clk rise edge)        0.000     0.000 r                       
                                                      0.000     0.000 r                       clk (IN)
                         net (fo=543, unset)          0.973     0.973                         ys[3].xs[2].torus_switch_xy/clk
    SLICE_X90Y70         FDRE                                         r  poly2_3_sw           ys[3].xs[2].torus_switch_xy/e_out_x_reg_reg[1]/C
  -------------------------------------------------------------------    ----------------------------------------
    SLICE_X90Y70         FDRE (Prop_fdre_C_Q)         0.518     1.491 f  poly2_3_sw           ys[3].xs[2].torus_switch_xy/e_out_x_reg_reg[1]/Q
                         net (fo=4, routed)           1.492     2.983                         ys[3].xs[3].torus_switch_xy/s0/e[2][3][3]
    SLICE_X67Y70         LUT5 (Prop_lut5_I3_O)        0.124     3.107 f  poly3_3_sw           ys[3].xs[3].torus_switch_xy/s0/o_b_r_i_3__14/O
                         net (fo=9, routed)           0.913     4.020                         ys[3].xs[3].torus_switch_xy/s0/o_b_r_i_3__14_n_0
    SLICE_X66Y70         LUT3 (Prop_lut3_I1_O)        0.117     4.137 r  poly3_3_sw           ys[3].xs[3].torus_switch_xy/s0/waiting_for_ack_i_6__13/O
                         net (fo=6, routed)           0.661     4.798                         ys[3].xs[3].torus_switch_xy/s0/o_b_r_reg_2
    SLICE_X66Y70         LUT5 (Prop_lut5_I3_O)        0.348     5.146 r  poly3_3_sw           ys[3].xs[3].torus_switch_xy/s0/e_out_y_reg[1]_i_1__14/O
                         net (fo=1, routed)           0.899     6.045                         ys[3].xs[3].torus_switch_xy/e_msg[33]
    SLICE_X53Y70         FDRE                                         r  poly3_3_sw           ys[3].xs[3].torus_switch_xy/e_out_y_reg_reg[1]/D
  -------------------------------------------------------------------    ----------------------------------------

                         (clock clk rise edge)        6.000     6.000 r                       
                                                      0.000     6.000 r                       clk (IN)
                         net (fo=543, unset)          0.924     6.924                         ys[3].xs[3].torus_switch_xy/clk
    SLICE_X53Y70         FDRE                                         r  poly3_3_sw           ys[3].xs[3].torus_switch_xy/e_out_y_reg_reg[1]/C
                         clock pessimism              0.000     6.924                           
                         clock uncertainty           -0.035     6.889                           
    SLICE_X53Y70         FDRE (Setup_fdre_C_D)       -0.058     6.831    poly3_3_sw             ys[3].xs[3].torus_switch_xy/e_out_y_reg_reg[1]
  -------------------------------------------------------------------
                         required time                          6.831                           
                         arrival time                          -6.045                           
  -------------------------------------------------------------------
                         slack                                  0.786                           




