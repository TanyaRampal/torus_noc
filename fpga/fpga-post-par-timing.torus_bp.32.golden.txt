Copyright 1986-2022 Xilinx, Inc. All Rights Reserved.
-----------------------------------------------------------------------------------------
| Tool Version : Vivado v.2022.1 (lin64) Build 3526262 Mon Apr 18 15:47:01 MDT 2022
| Date         : Tue Nov 26 14:02:52 2024
| Host         : ECEUBUNTU2 running 64-bit Red Hat Enterprise Linux release 8.10 (Ootpa)
| Command      : report_timing -file fpga-post-par-timing.torus_bp.32.txt
| Design       : torus_bp
| Device       : 7z020-clg400
| Speed File   : -1  PRODUCTION 1.12 2019-11-22
-----------------------------------------------------------------------------------------

Timing Report

Slack (MET) :             2.209ns  (required time - arrival time)
  Source:                 ys[1].xs[2].switch/e_out_r_reg[35]/C
                            (rising edge-triggered cell FDRE clocked by clk  {rise@0.000ns fall@4.500ns period=9.000ns})
  Destination:            ys[1].xs[3].cli/packet_num_reg[28]/CE
                            (rising edge-triggered cell FDRE clocked by clk  {rise@0.000ns fall@4.500ns period=9.000ns})
  Path Group:             clk
  Path Type:              Setup (Max at Slow Process Corner)
  Requirement:            9.000ns  (clk rise@9.000ns - clk rise@0.000ns)
  Data Path Delay:        6.299ns  (logic 1.239ns (19.671%)  route 5.060ns (80.329%))
  Logic Levels:           4  (LUT5=3 LUT6=1)
  Clock Path Skew:        -0.049ns (DCD - SCD + CPR)
    Destination Clock Delay (DCD):    0.924ns = ( 9.924 - 9.000 ) 
    Source Clock Delay      (SCD):    0.973ns
    Clock Pessimism Removal (CPR):    0.000ns
  Clock Uncertainty:      0.035ns  ((TSJ^2 + TIJ^2)^1/2 + DJ) / 2 + PE
    Total System Jitter     (TSJ):    0.071ns
    Total Input Jitter      (TIJ):    0.000ns
    Discrete Jitter          (DJ):    0.000ns
    Phase Error              (PE):    0.000ns

    Location             Delay type                Incr(ns)  Path(ns)    PBlock               Netlist Resource(s)
  -------------------------------------------------------------------    ----------------------------------------
                         (clock clk rise edge)        0.000     0.000 r                       
                                                      0.000     0.000 r                       clk (IN)
                         net (fo=3647, unset)         0.973     0.973                         ys[1].xs[2].switch/clk
    SLICE_X96Y90         FDRE                                         r  poly2_1_sw           ys[1].xs[2].switch/e_out_r_reg[35]/C
  -------------------------------------------------------------------    ----------------------------------------
    SLICE_X96Y90         FDRE (Prop_fdre_C_Q)         0.518     1.491 f  poly2_1_sw           ys[1].xs[2].switch/e_out_r_reg[35]/Q
                         net (fo=3, routed)           2.123     3.614                         ys[1].xs[3].switch/s0/s_d_r_reg[35]_0[35]
    SLICE_X51Y89         LUT5 (Prop_lut5_I0_O)        0.124     3.738 f  poly3_1_sw           ys[1].xs[3].switch/s0/s_d_r[35]_i_3__5/O
                         net (fo=6, routed)           0.714     4.452                         ys[1].xs[3].switch/s0/s_d_r[35]_i_3__5_n_0
    SLICE_X52Y92         LUT5 (Prop_lut5_I0_O)        0.118     4.570 r  poly3_1_sw           ys[1].xs[3].switch/s0/s_out_r[35]_i_2__6/O
                         net (fo=6, routed)           0.720     5.289                         ys[1].xs[3].cli/regulator/i_y_r_reg[0]_0
    SLICE_X54Y92         LUT6 (Prop_lut6_I2_O)        0.326     5.615 r  poly3_1_cli          ys[1].xs[3].cli/regulator/waiting_for_ack_i_4__6/O
                         net (fo=10, routed)          0.644     6.259                         ys[1].xs[3].cli/regulator/o_b_r_reg_inv
    SLICE_X55Y93         LUT5 (Prop_lut5_I4_O)        0.153     6.412 r  poly3_1_cli          ys[1].xs[3].cli/regulator/waiting_for_ack_i_2__6/O
                         net (fo=33, routed)          0.859     7.272                         ys[1].xs[3].cli/consume
    SLICE_X57Y100        FDRE                                         r  poly3_1_cli          ys[1].xs[3].cli/packet_num_reg[28]/CE
  -------------------------------------------------------------------    ----------------------------------------

                         (clock clk rise edge)        9.000     9.000 r                       
                                                      0.000     9.000 r                       clk (IN)
                         net (fo=3647, unset)         0.924     9.924                         ys[1].xs[3].cli/clk
    SLICE_X57Y100        FDRE                                         r  poly3_1_cli          ys[1].xs[3].cli/packet_num_reg[28]/C
                         clock pessimism              0.000     9.924                           
                         clock uncertainty           -0.035     9.889                           
    SLICE_X57Y100        FDRE (Setup_fdre_C_CE)      -0.408     9.481    poly3_1_cli            ys[1].xs[3].cli/packet_num_reg[28]
  -------------------------------------------------------------------
                         required time                          9.481                           
                         arrival time                          -7.272                           
  -------------------------------------------------------------------
                         slack                                  2.209                           




